go.mod

```diff
@@ -1,3 +1,25 @@
 module github.com/gofiber/storage
 
 go 1.17
+
+require (
```

Comment:
Why this did go.mod get modified? It should only be the one under `coundbase/go.mod`

Reply:
It was modified unintentially. I rewrote the git commit history and force pushed new commit that removes modifications on base go.mod file 👍